packages/app-project/stores/YourStats.js

```diff
@@ -0,0 +1,94 @@
+import { autorun } from 'mobx'
+import { addDisposer, flow, getRoot, types } from 'mobx-state-tree'
+import auth from 'panoptes-client/lib/auth'
+import asyncStates from '@zooniverse/async-states'
+import { panoptes } from '@zooniverse/panoptes-js'
+
+const YourStats = types
+  .model('YourStats', {
+    dailyCounts: types.optional(types.array(types.frozen({})), []),
+    error: types.maybeNull(types.frozen({})),
+    loadingState: types.optional(types.enumeration('state', asyncStates.values), asyncStates.initialized),
+    totalCount: 0
```

Comment:
This should probably be `types.optional(types.number, 0)`

Reply:
No problem. I think I copied this from the MST Getting Started docs, so I’ve been assuming it sets the type for you when you declare a default value.

Reply:
Turns out I should have read further on in the MST tutorial.
https://github.com/mobxjs/mobx-state-tree/blob/master/docs/getting-started.md#meeting-types

From reading that, I think `totalCount: 0` is a shortcut for `totalCount: type.optional(types.number, 0)` but the latter is better for clarity?